sample-plugin/pom.xml

```diff
@@ -162,13 +161,6 @@
             <groupId>io.jenkins.plugins</groupId>
             <artifactId>okhttp-api</artifactId>
             <scope>test</scope>
-            <exclusions>
-                <!-- TODO conflict with configuration-as-code -->
-                <exclusion>
-                    <groupId>org.jetbrains</groupId>
-                    <artifactId>annotations</artifactId>
-                </exclusion>
-            </exclusions>
```

Comment:
No longer needed after https://github.com/jenkinsci/okhttp-api-plugin/pull/144 and its subsequent adoption in this repository.